kino-nextjs/app/components/reviews/ReviewsList.tsx

```diff
@@ -8,37 +8,40 @@ import Link from 'next/link';
 
 interface ReviewsListProps {
   id: string;
-  searchParams?: {
-    page?: string;
-  };
+  page?: string;
 }
 
-export default async function ReviewsList({
-  id,
-  searchParams,
-}: ReviewsListProps) {
-  let page = Number(searchParams?.page) || 1;
-  page = !page || page < 1 ? 1 : page;
+export default async function ReviewsList({ id, page }: ReviewsListProps) {
+  let currentPage = Number(page) || 1;
+  currentPage = !currentPage || currentPage < 1 ? 1 : currentPage;
 
   const perPage = 5;
-  const { reviews, totalReviews } = await fetchReviews(id, page, perPage);
+  const { reviews, totalReviews } = await fetchReviews(
+    id,
+    currentPage,
+    perPage
+  );
 
   const totalPages = Math.ceil(totalReviews / 5);
 
-  const prevPage = page - 1 ? page - 1 : 1;
-  const nextPage = page + 1;
+  const prevPage = currentPage - 1 ? currentPage - 1 : 1;
+  const nextPage = currentPage + 1;
 
   const pageNumbers = [];
   const offsetNumber = 3;
-  for (let i = page - offsetNumber; i <= page + offsetNumber; i++) {
+  for (
+    let i = currentPage - offsetNumber;
+    i <= currentPage + offsetNumber;
+    i++
+  ) {
     if (i >= 1 && i <= totalPages) {
       pageNumbers.push(i);
     }
   }
 
   return (
     <>
-      <div className='flex flex-col rounded'>
+      <div data-cy={'review-list'} className='flex flex-col rounded'>

```

Comment:
Is data.cy insetead of classnames?

Reply:
> Is data.cy insetead of classnames?

Exactly! The data-cy attribute isolates them from other selectors and makes it easier to target elements :)